pkg/feature/cert.go

```diff
@@ -15,11 +16,46 @@ import (
 
 	"github.com/pkg/errors"
 	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	v1 "github.com/opendatahub-io/opendatahub-operator/v2/apis/dscinitialization/v1"
```

Comment:
We should stick to the convention everywhere else and name it `dsciv1`.

Reply:
Well, this is the editor doing the import for me :-)

It is changed.